packages/babel-plugin-transform-es2015-block-scoping/README.md

```diff
@@ -15,9 +15,19 @@ npm install --save-dev babel-plugin-transform-es2015-block-scoping
 **.babelrc**
 
 ```json
+// without options
```

Comment:
Sorry, but since were using json now, can we move the without options outside like in https://github.com/babel/babel/pull/5194/files?